### PR TITLE
meta tags added

### PR DESCRIPTION
**Pull request by ragupathis**
Meta tags for link preview added here,

Python logo used as a placeholder.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -3,6 +3,30 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+<!-- Open Graph Meta -->
+
+    <meta property="og:title" content="PySangamam - Tamil Nadu's first Python conference" />
```
Comment (astronomersiva) [inline _includes/head.html]: I guess this can be in title case. `Tamil Nadu's First Python Conference`

File: _includes/head.html [hunk 2]
```diff
@@ -3,6 +3,30 @@
...
+    <meta property="og:description" content="September 7th and 8th, 2018 IC & SR - IIT Madras " />
```
Comment (astronomersiva) [inline _includes/head.html]: The space after Madras can be removed.

File: _includes/head.html [hunk 3]
```diff
@@ -3,6 +3,30 @@
...
+    <meta property="og:url" content="http://pysangamam.org/" />
```
Comment (astronomersiva) [inline _includes/head.html]: @shrayasr Did you look into that HTTPS fix? If that is done, we can change this to https

File: _includes/head.html [hunk 4]
```diff
@@ -3,6 +3,30 @@
...
+    <meta property="og:image" content="img/python_logo.png" />
```
Comment (astronomersiva) [inline _includes/head.html]: It's a convention to have hyphenated file names in website assets.

File: _includes/head.html [hunk 5]
```diff
@@ -3,6 +3,30 @@
...
+      <link itemprop="url" href="img/python_logo.png">
+    </span>
+
+  <!-- Twitter Summary -->
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@PySangamam" />
+    <meta name="twitter:creator" content="@PySangamam" />
+
+
+    <meta name="description" content="PySangamam - Tamil Nadu's first Python conference - September 7th and 8th, 2018 IC & SR - IIT Madras" />
```
Comment (astronomersiva) [inline _includes/head.html]: Title case here also.

File: _includes/head.html [hunk 6]
```diff
@@ -3,6 +3,30 @@
...
+    <meta name="keywords" content="PySangamam, Python, ChennaiPy" />
```
Comment (astronomersiva) [inline _includes/head.html]: Can we add 'Conference' to this?

**Review by astronomersiva — CHANGES_REQUESTED** [review]

File: _includes/head.html (hunk 1 above)
Comment (astronomersiva) [reply _includes/head.html]: @shrayasr ping 🔔

File: _includes/head.html (hunk 6 above)
Comment (ragupathis) [reply _includes/head.html]: Added

**Review by astronomersiva — APPROVED** [review]

**Comment by astronomersiva** [comment]
Awesome work @ragupathis !

**Pull request merged**